pkgs/os-specific/linux/roccat-tools/default.nix

```diff
@@ -0,0 +1,112 @@
+{ stdenv, fetchurl, cmake, dbus, glib, libcanberra, atk, cairo
+, gdk_pixbuf, gtk2, pango, libnotify, libgudev, xorg, sqlite
+, gegl, dbus_glib, gettext, pkgconfig, shadow, devices ? false
+}:
```

Comment:
I could probably do this if I find the time.
However, concatStringsSep really should be documented properly if this derivation is to be easier to parse (for humans.)